Av1an/bar.py

```diff
@@ -34,13 +34,11 @@ def update(self, value):
 BaseManager.register('Counter', Counter)
 
 
-def make_pipes(ffmpeg_gen, command):
+def make_pipes(ffmpeg_gen_cmd: list, command: list):
```

Comment:
`ffmpeg_gen_cmd` is now a `Tuple[str]`.

`command` is now a tuple containing a list and a tuple `(['ffmpeg', ...], ('aomenc', ...))`. `command` should contain only 1 type inside of it. Pick either `Tuple[List[str]]` or `Tuple[Tuple[str]]`. Since it seems like most commands are going to be tuples, I would prefer `Tuple[Tuple[str]]`.

```suggestion
def make_pipes(ffmpeg_gen_cmd: Tuple[str], command: Tuple[Tuple[str]]):
```